src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,8 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final class Intake{
+        public static final double TICK_PER_METERS = 0;
```

Comment:
TICKS_PER_METER! not TICK_PER_METERS

Reply:
Why do you need this constant for the intake?